lib/resty/requests/response.lua

```diff
@@ -312,9 +313,7 @@ local function json(r)
     end
 
     content_type = lower(content_type)
-    if content_type ~= "application/json"
-       and content_type ~= "application/json; charset=utf-8"
-    then
+    if not includes(content_type, "application/json") then
```

Comment:
Just use `string.find` is enough to satisfy this demand.

Reply:
It seems you are right. See the issue first #29 , i have left some messages.